.github/workflows/test.yml

```
@@ -110,3 +110,45 @@ jobs:
       - name: Check .gitconfig
         run: |
           cat ~/.gitconfig
+
+      - name: Test do not retry if command successfull
+        run: |
+          source try.sh
+
+          before=$(date +%s)
+          try_till_success ls
+          after=$(date +%s)
+
+          # コマンド呼び出しに時間がかかっているかでリトライ判定をしています
+          if [ $(expr $after - $before) -gt 1 ]; then
```

```suggestion
          if [ $(($after - $before)) -gt 1 ]; then
```
actionlint 的にはこう言われそう

---

1秒以上は失敗するとリトライで 1秒以上かかる(1秒 sleep を3回，実際には exponential back off でもうちょっとかかりますが)のを判定してるって感じですね．

---

そうです！
suggestion そのまま取り込んじゃいますね